com.civicrm.hrjobroles/js/test/services/date-validation.service.spec.js

```diff
@@ -102,19 +102,19 @@ define([
     });
 
     it('Start date cannot be higher', function () {
-            /* Should pass */
+      /* Should pass */
       expect(function () {
         dateValidation.validate('12/11/2003', '10/01/2009');
       }).not.toThrow();
 
-            /*  Should NOT pass. Start date cannot be higher */
+      /*  Should NOT pass. Start date cannot be higher */
```

Comment:
I am confused with these comments. What does "should not pass" mean in this case? Does it mean "should throw an exception"? If yes, I suggest to get rid of these comments, it is clear from the code what should happen.

Same for other places